docs/pricing-structures.md

```diff
@@ -0,0 +1,39 @@
+## Pricing Structures - For Cypherpunktimes.com Proposal 2024-25
+This document details the pricing structures for each participant in the cypherpunktimes.com proposal. We have now removed the editor role, art and technical articles from this proposal. This will help standardise payments across the proposal and provide greater transparency. 
+
+There remains opportunities for creating advanced content like technical articles and art, but as these tend to be specialist in nature, it makes sense to provide separate proposals for this content.
+
+## Article pricing
+
+As detailed in the process document (https://github.com/PhoenixGreen/Cypherpunk-Times/blob/main/docs/process.md) cypherpunktimes.com has now moved to a fixed payment model for producing paid for articles, which consist of four independent stages and one optional stage.
+
+The aim of this structure is to incentivise high quality written content, whilst standardising author payments. Writers are now incentivised to participate in planning, discussion, peer review, and social media engagement.
+
+As part of this process, and to allow for more time on each stage, we are limiting each writer to a maximum of two articles per month. 
+
+An approved article now has the associated stages and hourly rate applied to it:
+
+Stage 1: Discussion and planning = 1hr
+Stage 2: Create article = 5hrs
+Stage 3: Review = 1hr
+Stage 4: Social media = 1hr
+Optional Stage: TwitterSpace / Livestream = 3hrs
+
+## Official reviewer pricing
+
+As detailed in the process document (https://github.com/PhoenixGreen/Cypherpunk-Times/blob/main/docs/process.md) reviewers are now responsible for doing quality checks and giving constructive feedback for possible improvements. The aim of the review process is that each article will have one or more reviewers conduct planning, pre-review and review checks.
+
+Maximum payment for planning and pre-review checks = 1 to 2hr
+Maximum payment for article review checks = 2 to 4hrs
```

Comment:
Good to see these mentioned explicitly. May incentivize more reviewers to join the process and improve the output.

I would clarify if these limits are *per reviewer* or *per article*. If they are per reviewer, we will need to also add a maximum amount of reviewers who can get paid. If they are per article, I would clarify that the author/admin can involve as many reviewers as they like as long as the total payment fits the limit.

Reply:
Really good points. Initially, I'm thinking maximum billing per article with maximum of two reviewers. 
e.g. if two reviewers get involved they work to the limit of 2hrs each for the review stage.

I'm happy to take advice on this. You've got more experience in this area, how do you think I should structure it? Do you think my maximums is high enough? All ideas greatly appreciated.

Reply:
Up to 4 hours and up to 2 reviewers sounds good.

From my experience you cannot perfectly guess these limits ahead of time, so just pick something reasonable, don't unnecessarily limit yourself, and adjust as needed later.